src/components/editor/editor-pane/autocompletion/image.ts

```diff
@@ -0,0 +1,40 @@
+import { Editor, Hint, Hints, Pos } from 'codemirror'
+import { findWordAtCursor, Hinter } from './index'
+
+const allowedChars = /(!|\[|]])*/
+const wordRegExp = /^(!(\[.*])?)$/
+const allSupportedImages = [
+  '![image alt](https://example.com)',
+  '![image alt](https://example.com =WidthxHeight)',
```

Comment:
Even if the title-attribute is currently not yet implemented, we maybe want to include it here as in 1.6?

```suggestion
  '![image alt](https:// "title")',
  '![image alt](https:// "title" =WidthxHeight)',
```

Reply:
done